urg_node:
  ros__parameters:
    #ip_address: ""
    ip_port: 10940
    laser_frame_id: "laser"
    serial_port: "/dev/ttyACM0"
    serial_baud: 115200
    calibrate_time: false
    publish_intensity: false
    publish_multiecho: false
    error_limit: 4
    diagnostics_tolerance: 0.05
    diagnostics_window_time: 5.0
    get_detailed_status: false
    default_user_latency: 0
    angle_min: -3.14
    angle_max: 3.14